hw/ip/spi_device/doc/_index.md

```diff
@@ -31,17 +44,24 @@ peripheral pin SCK. In this design the SCK is directly used to drive the
 interface logic as its primary clock, which has performance benefits, but incurs
 design complications described later.
 
+The SW can receive TPM commands with payload (address and data) and respond to the read commands with the return data with the TPM submodule in the SPI_DEVICE HWIP.
```

Comment:
with the return data with the tpm submodule -> with the return data using the tpm submodule i think sounds a bit better.

Reply:
Thanks!